lib/katello/engine.rb

```diff
@@ -87,6 +87,13 @@ def find_assets(args = {})
       require 'katello/apipie/validators'
     end
 
+    # make sure the Katello plugin is initialized before `after_initialize`
+    # hook so that the resumed Dynflow tasks can rely on everything ready.
+    initializer 'katello.register_plugin', :before => :finisher_hook do
```

Comment:
In fact, in https://github.com/theforeman/foreman-tasks/pull/185, the initialization of dynflow runtime was moved to after_initialize - not the dynflow configuration, that can be performed in any initializer block as before, so I've replaced the occurrences of `:before => 'foreman_tasks.initialize_dynflow'` with `:before => :finisher_hook`.